### DISCUSSION TITLE
Column select when ctrl is pressed first

### DISCUSSION THREAD

**Discussion by mh013** [unixwork/xnedit#150] [Ideas]
Good day,
first of all, thank you for your amazing work with nedit. It has been by favorite editor for almost 20 years and you saved it from extinction, thank you!
The original nedit implementation had the multi-column selection enabled regardless of the order the ctrl/command button is pressed with respect to the mouse click/drag. However, that's not the case with the xnedit. Xnedit requires the mouse click and drag to be done first then the ctrl button to be pressed in order to activate the multi-column select.
I suggest to change the multi-column selection behavior to be true to the original nedit functionality if possible.

Once again, really appreciate your time, dedication and amazing work.
Thank you!

**Comment by unixwork** [comment]
Hi,

on my system, pressing ctrl first still works. What OS and Desktop Environment are you using?

Also, can you check some older versions, for example xnedit 1.4? Would be interesting to know, which commit changed the behavior. Version 1.5 introduced the new multicursor feature, which required some changes in input handling, maybe that is the cause of this bug.

**Reply by mh013** [reply]
Thank you for your prompt reply and sorry for the late one on my side.
I confirm that the issue isn't from xnedit, turns out my rdp client is the culprit. It works fine if I work on a native machine, but when I connect to same machine through rdp the issue appears.
Thanks you :)
